src/pages/Profile.tsx

```diff
@@ -1,48 +1,98 @@
 import { useMetaMask } from "../contexts/MetaMaskContext";
 import Typography from "@mui/material/Typography";
-import Box from "@mui/material/Box";
+import {
+  StyledProfileWrapper,
+  StyledProfile,
+} from "./styledPages/Profile.styled";
 
 function Profile() {
   const { isMetamaskConnected, address, balance } = useMetaMask();
 
   return (
-    <Box textAlign="center">
-      <h2>Profile Page</h2>
-      {isMetamaskConnected ? (
-        <Typography
-          variant="body1"
-          align="center"
-          style={{ fontWeight: "bold", color: "black" }}
-        >
-          <p>Connected with address: {address}</p>
-          {balance && <p>Balance: {balance} ETH</p>}
+    <StyledProfileWrapper>
+      <StyledProfile>
+        <h2>Profile Page</h2>
+        <Typography variant="body1">
+          {isMetamaskConnected ? (
+            <>
+              Connected with address: {address}
+              {balance && (
+                <>
+                  <br />
+                  Balance: {balance} ETH
+                </>
+              )}
+            </>
+          ) : (
+            <>Please connect to MetaMask to view your profile.</>
```

Comment:
if i cant access profile page while i am not connected, when will i see this text???